scripts/travis-beforeinstall.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+set -x
+
+isFunction() {
```

Comment:
This script looks a bit overengineered. How about having two separate scripts so all this logic around argument parsing is not needed?

Reply:
Yop.

Reply:
Removed